HLTrigger/Muon/plugins/HLTMuonL1TFilter.cc

```diff
@@ -110,7 +111,13 @@ bool HLTMuonL1TFilter::hltFilter(edm::Event& iEvent,
       MuonRef muon(allMuons, distance(allMuons->begin(allMuons->getFirstBX()), it));
 
       // Only select muons that were selected in the previous level
-      if (find(prevMuons.begin(), prevMuons.end(), muon) == prevMuons.end())
+      bool matchPrevL1 = false;
+      int prevSize = prevMuons.size();
+      for (int it2 = 0; it2 < prevSize; it2++) {
+        if (deltaR2(muon->eta(), muon->phi(), prevMuons[it2]->eta(), prevMuons[it2]->phi()) < 0.1)
+          matchPrevL1 = true;
+      }
```

Comment:
```suggestion
        if (reco::deltaR2(muon->eta(), muon->phi(), prevMuons[it2]->eta(), prevMuons[it2]->phi()) < maxDeltaR2_){
          matchPrevL1 = true;
          break; 
        }
      }
```
Looks like the 0.1 qualifies as "magic number" here, so it would need to be replaced by a configuration parameter (held by a class member as in the suggested snippet). Also, is 0.1 the threshold for Delta-R or Delta-R^2 ?

Reply:
I have made the 0.1 configurable in the factorized PR. In the way it's currently used, it is basically arbitrary since the parameters of the L1 candidates that need to be matched are identical.